### ISSUE TITLE
error by using jaccard_loss

### ISSUE THREAD

**Issue by EnQing626** [kevinzakka/pytorch-goodies#6]
Very thanks for your code.
When I use the "jaccard_loss" function, some error happened:
`Too many indices for tensor of dimension 2` in line 110.(The num_classes  is 1)

My input size of true is 3*1*7*7 and the input size of logits is 3*1*7*7.
Can you help me to solve the problem~?
Thanks

**Comment by mfmezger** [comment]
Hi,

can you try changing the line for the true_1_hot, and adding a cast to int64.

Like this

`true_1_hot = torch.eye(num_classes)[true.squeeze(1).to(torch.int64)]`

**Comment by EnQing626** [comment]
> Hi,
> 
> can you try changing the line for the true_1_hot, and adding a cast to int64.
> 
> Like this
> 
> `true_1_hot = torch.eye(num_classes)[true.squeeze(1).to(torch.int64)]`

It works for me. Thanks for your reply :)
